src/CUDAdrv.jl

```diff
@@ -3,21 +3,24 @@ __precompile__()
 module CUDAdrv
 
 using Compat
-import Compat.String
+using Compat.String
```

Comment:
> `import` is the canonical way of only bringing only one name into the current namespace, so I would also prefer the old version.

Only for bringing a module name into the namespace without its exports. `import Mod` vs `using Mod` is somewhat backwards compared to `import Mod.func` vs `using Mod.func`. For functions, `import` allows extension and `using` doesn't, so using is safer.